example/server.mjs

```diff
@@ -1,29 +1,71 @@
+import fs from "node:fs";
+import path from "node:path";
 import fastify from "fastify";
-import { remixFastifyPlugin } from "@mcansh/remix-fastify";
+import { createRequestHandler, staticFilePlugin } from "@mcansh/remix-fastify";
 import { installGlobals } from "@remix-run/node";
+import sourceMapSupport from "source-map-support";
+import chokidar from "chokidar";
```

Comment:
i’d like to do a dynamic import here instead, but getting an error about not being able to dynamically import `node:events`